src/series.jl

```diff
@@ -21,18 +21,25 @@ convertToAnyVector(n::Integer, plotattributes::KW) = Any[zeros(0) for i in 1:n],
 
 # numeric vector
 convertToAnyVector(v::AVec{T}, plotattributes::KW) where {T<:Number} = Any[v], nothing
+convertToAnyVector(v::AVec{Union{Missing, T}}, plotattributes::KW) where {T<:Number} = Any[replace(v, missing => NaN)], nothing
 
 # string vector
 convertToAnyVector(v::AVec{T}, plotattributes::KW) where {T<:AbstractString} = Any[v], nothing
+convertToAnyVector(v::AVec{Union{Missing, T}}, plotattributes::KW) where {T<:AbstractString} = Any[replace(v, missing => "")], nothing
 
 function convertToAnyVector(v::AMat, plotattributes::KW)
+    v = handlemissingsinmatrix(v)
     if all3D(plotattributes)
         Any[Surface(v)]
     else
         Any[v[:,i] for i in 1:size(v,2)]
     end, nothing
 end
 
+handlemissingsinmatrix(v::AMat) = v
```

Comment:
Why not just `handlemissings`?

Reply:
no reason, I guess just to signal that it's a very local function just for use above. Happy to rename.

Reply:
done

Reply:
Great, I would have been ok with `handlemissindsinmatrix` too though. I was just curious and wondering, if you were also planning to implement something like `handlemissingsinvector` or similar too, and in that case I would have suggested dispatching on the types.